.github/unittest/action.yml

```diff
@@ -0,0 +1,26 @@
+name: Run .NET unit tests
+description: Runs .NET unit tests under a specified configuration
+
+inputs:
+  project:
+    default: ""
+    description: The test project
+    required: true
+  configuration:
+    default: Debug
+    description: The build configuration
+    required: false
+
+runs:
+  using: composite
+  steps:
+  - run: dotnet msbuild "${{ inputs.project }}"
```

Comment:
actually what's the difference between dotnet build and dotnet msbuild in this case?

Reply:
None. Fixed. :D